infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -75,11 +75,13 @@ type OfflineStore struct {
 }
 
 // OfflineStorePersistence configures the persistence settings for the offline store service
+// +kubebuilder:validation:XValidation:rule="[has(self.file), has(self.store)].exists_one(c, c)",message="One selection required."
```

Comment:
can we add more details to the message like `between file or store`?

Reply:
Done